api/src/main/java/org/pmiops/workbench/actionaudit/auditors/EgressEventAuditorImpl.kt

```diff
@@ -19,25 +15,29 @@ import org.pmiops.workbench.db.dao.UserDao
 import org.pmiops.workbench.db.dao.WorkspaceDao
 import org.pmiops.workbench.db.model.DbEgressEvent
 import org.pmiops.workbench.exceptions.BadRequestException
-import org.pmiops.workbench.model.EgressEvent
-import org.pmiops.workbench.model.EgressEventRequest
+import org.pmiops.workbench.model.SumologicEgressEvent
+import org.pmiops.workbench.model.SumologicEgressEventRequest
 import org.pmiops.workbench.workspaces.WorkspaceService
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.stereotype.Service
+import java.time.Clock
+import java.util.Optional
+import java.util.logging.Logger
+import javax.inject.Provider
 
 @Service
 class EgressEventAuditorImpl @Autowired
 constructor(
-    private val actionAuditService: ActionAuditService,
-    private val workspaceService: WorkspaceService,
-    private val workspaceDao: WorkspaceDao,
-    private val userDao: UserDao,
-    private val clock: Clock,
-    @Qualifier("ACTION_ID") private val actionIdProvider: Provider<String>
+        private val actionAuditService: ActionAuditService,
```

Comment:
nit, format was changed, you might want to set Kotlin to also use GoogleStyle schema

Reply:
fixed with spotless - yeah I probably need to adjust my intellij Kotlin settings